### PR TITLE
governance examples

### PR DESCRIPTION
**Pull request by levackt**
Updated the titles to be more suggestive. Although it only tripped me up, maybe it helps future me anyway.

### CODE REVIEW

File: docs/guides/governance.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ Where `proposal.json` is:
 
 ```json
 {
-  "title": "Community Pool Spend",
+  "title": "Community Pool Spend: My Cool Proposal",
```
Comment (toml01) [inline docs/guides/governance.md:160]: If we're trying to make it more suggestive, maybe something like "My cool dApp" or "Super important committee" would be better in this case?

Comment (levackt) [reply docs/guides/governance.md]: Sounds good to me

File: docs/guides/governance.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ Where `proposal.json` is:
 ```json
 {
   "type": "Text",
-  "title": "My Cool Proposal",
+  "title": "Community Pool Spend: Super important committee",
```
Comment (toml01) [inline docs/guides/governance.md:55]: Now it says `Community Pool Spend` on a text prop. Is this on purpose?

Comment (levackt) [reply docs/guides/governance.md]: No sorry, fixed. Thanks Tom

**Review by toml01 — APPROVED** [review]

**Pull request merged**